pkgs/development/python-modules/spacy/default.nix

```diff
@@ -37,7 +37,8 @@ buildPythonPackage rec {
       --replace "ftfy==" "ftfy>=" \
       --replace "msgpack-python==" "msgpack-python>=" \
       --replace "msgpack-numpy==" "msgpack-numpy>=" \
-      --replace "pathlib" "pathlib; python_version<\"3.4\""
+      --replace "thinc>=6.10.3,<6.11.0" "thinc>=6.10.3,<6.12.0" \
+      --replace "plac<1.0.0,>=0.9.6" "plac<1.1.0,>=0.9.6"
```

Comment:
Why not removing the upper bound?

Reply:
I could do that, but I am not sure how much these packages stick to semver or any API stability for that matter ;). I did some small tests to confirm that plac 1.0 and thinc 6.11 work with spacy.

But I'll update the PR without these upper bounds.